internal/services/pubkey_service.go

```diff
@@ -49,7 +50,10 @@ func CreatePubkey(w http.ResponseWriter, r *http.Request) {
 func ListPubkeys(w http.ResponseWriter, r *http.Request) {
 	pubkeyDao := dao.GetPubkeyDao(r.Context())
 
-	pubkeys, err := pubkeyDao.List(r.Context(), 100, 0)
+	offset := pagination.GetOffset(r.Context())
+	limit := pagination.GetLimit(r.Context())
```

Comment:
Maybe Offset64/Limit64 too? :-D

Reply:
Do you mean to do the conversion here like that:
 ```
int64(pagination.GetOffset(r.Context())) 
```
or to return int64 from the Offset function?

Reply:
Just an idea (you don’t have to implement it if you don’t like it):

    func GetOffset64(context.Context) int64